src/components/common/NewMatch/index.js

```diff
@@ -0,0 +1,62 @@
+import smilies from 'assets/smilies.png';
+import defaultProfile from 'assets/default-profile.png';
+import useTranslation from 'hooks/useTranslation';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectTab, setHomeTab, setNewMatch } from 'state/slices/tabSlice';
+import { setConversationSelected } from 'state/slices/targetSlice';
+import Button from '../Button';
+import { tabsEnum } from '../Tabs';
+import './styles.scss';
+
+const NewMatch = () => {
+  const t = useTranslation();
+  const dispatch = useDispatch();
+  const { newMatch } = useSelector(selectTab);
+
+  if (!newMatch) return null;
+
+  const onStart = () => {
+    dispatch(setConversationSelected(newMatch.match_conversation?.id));
+    dispatch(setHomeTab(tabsEnum.chat));
+    dispatch(setNewMatch(null));
+  };
+
+  const onSkip = () => dispatch(setNewMatch(null));
+
+  return (
+    <>
+      <div className="new-match-dialog">
```

Comment:
WDYT if you replace all those divs tags with some semantic HTML tags? Like section or article?

Reply:
Will look into it! I definitely overuse divs as a one trick pony